src/library/rssscanner.c

```diff
@@ -453,6 +453,13 @@ rss_save(struct playlist_info *pli, int *count, enum rss_scan_type scan_type)
 	  if (ret > 0)
 	    continue;
 	}
+      else if (scan_type == RSS_SCAN_META)
+        {
```

Comment:
The indentation is still incorrect - compare with above

Reply:
This is git diff mis-displaying tabs (the `RSS_SCAN_RESCAN` block) vs 8 spaces here - when I look at it in a terminal the indentation is correct.  Looking at other files - from other authors - theres also this mix of 8spaces vs tabs as well as this file.  Let me re-push this as tabs for this so it looks correct in the gibthub diff and can merge
![tabsvsspaces](https://user-images.githubusercontent.com/18466811/79549629-2654ed00-808f-11ea-870c-7e39a948362e.png)

Reply:
Historically, forked-daapd has a special indentation style, and 8 spaces is incorrect. So it is not due to git diff. It is quite possible that others have made the same error.